Library/Homebrew/cmd/audit.rb

```diff
@@ -659,6 +659,25 @@ def audit_line(line, lineno)
     end
   end
 
+  def audit_prefix_has_contents
+    return unless formula.prefix.directory?
+
+    unless Dir.glob("#{formula.prefix}/**/*").any? do |f|
```

Comment:
Re-read the code as written in this pull request. There is no code inside the `unless`.

In other words

```rb
unless (a bunch of code)
end
```

is exactly the same as

```rb
(a bunch of code)
```

The unless is not doing anything here. There is no code inside of it.

Are you missing an `end` somewhere in the middle?